src/components/ProductList.jsx

```
@@ -19,14 +20,35 @@ function ProductList() {
+      console.log(apiResponse.data);
+      fetchProducts();
+    } catch (e) {
+      throw new Error(e);
+    }
   };
 
   const handleEdit = async (updatedProduct) => {
     // TODO: PUT API를 호출하고 fetchProducts() 호출
+    try {
+      console.log(updatedProduct.id);
+      const apiResponse = await instance.put(updatedProduct.id, updatedProduct);
```

ESLint

[error] 'apiResponse' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)